pkg/features/kube_features.go

```diff
@@ -409,6 +409,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	genericfeatures.APIResponseCompression:  {Default: false, PreRelease: utilfeature.Alpha},
 	genericfeatures.Initializers:            {Default: false, PreRelease: utilfeature.Alpha},
 	genericfeatures.APIListChunking:         {Default: true, PreRelease: utilfeature.Beta},
+	genericfeatures.DryRun:                  {Default: false, PreRelease: utilfeature.Alpha},
```

Comment:
@sttts do we have a apiserver specific feature flags yet?  I really don't want this one leaking into clients.  It seems ripe for naughtyness

Reply:
I can rename it if it helps? `ServerSideDryRun`?

Reply:
+1 for the rename.